pkgs/development/compilers/rust/1_82.nix

```diff
@@ -103,25 +103,23 @@ import ./default.nix
       else
         llvmPackages_18;
 
-    # Note: the version MUST be one version prior to the version we're
-    # building
```

Comment:
It may be worth writing a similar comment here, e.g. "the version must be the same version that we're building".

Reply:
Good point. Done.